packages/blade/scripts/generateGitHubRegistryNpmrc.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Context about this script:
+ *
+ * we want to publish to our internal GitHub regitry as well as public npm registry and since we can only have 1 npmrc
+ * at a time and we only need it while publishing to registries so we'll generate it with this script for publishing to
+ * our internal GitHub registry and to publish to public registry we do it from `./publishToNpm.js`
+ *
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const BLADE_ROOT = path.join(__dirname, '..');
+const NPMRC_PATH = path.join(BLADE_ROOT, '.npmrc');
```

Comment:
Do you foresee any changes for creating `.npmrc` in `./packages/blade/.npmrc` instead of the existing `./.npmrc`?

For npm I was putting it in blade directory because I was running publish inside blade directory 🤔 

So will this flow work with changeset?

Reply:
oh yeah good catch i'll change this logic

Reply:
fixed [7031215](https://github.com/razorpay/blade/pull/799/commits/7031215ea7543abf7ad20dc8ab44d8c26e7466fd)